examples/mapillarystreetview.js

```diff
@@ -0,0 +1,109 @@
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import './streetview.css';
+import angular from 'angular';
+import olMap from 'ol/Map.js';
+
+import olView from 'ol/View.js';
+import olLayerTile from 'ol/layer/Tile.js';
+import olSourceOSM from 'ol/source/OSM.js';
+import olStyleFill from 'ol/style/Fill.js';
+import olStyleStroke from 'ol/style/Stroke.js';
+import olStyleStyle from 'ol/style/Style.js';
+import olStyleText from 'ol/style/Text.js';
+import ngeoStreetviewModule from 'ngeo/streetview/module.js';
+import ngeoMapModule from 'ngeo/map/module.js';
+import ngeoMiscToolActivate from 'ngeo/misc/ToolActivate.js';
+import ngeoMiscToolActivateMgr from 'ngeo/misc/ToolActivateMgr.js';
+
+/** @type {angular.IModule} **/
+const module = angular.module('app', [
+  'gettext',
+  ngeoMapModule.name,
+  ngeoStreetviewModule.name,
+  ngeoMiscToolActivateMgr.name,
+]);
+
+/**
+ * @param {import("ngeo/map/FeatureOverlayMgr.js").FeatureOverlayMgr} ngeoFeatureOverlayMgr
+ *    Ngeo FeatureOverlay manager.
+ * @param {import("ngeo/misc/ToolActivateMgr.js").ToolActivateMgr} ngeoToolActivateMgr
+ *    Ngeo ToolActivate manager service.
+ * @constructor
+ * @ngInject
+ */
+function MainController(ngeoFeatureOverlayMgr, ngeoToolActivateMgr) {
+  /**
+   * @type {import("ol/style/Style.js").default}
+   */
+  this.style = new olStyleStyle({
+    text: new olStyleText({
+      fill: new olStyleFill({color: '#279B61'}),
+      font: '900 30px "Font Awesome 5 Free"',
+      offsetY: -15,
+      stroke: new olStyleStroke({color: '#ffffff', width: 3}),
+      text: '\uf041',
+    }),
+  });
+
+  /**
+   * @type {import("ol/Map.js").default}
+   */
+  this.map = new olMap({
+    layers: [
+      new olLayerTile({
+        source: new olSourceOSM(),
+      }),
+    ],
+    view: new olView({
+      center: [830604, 5932807],
+      zoom: 17,
+    }),
+  });
+
+  ngeoFeatureOverlayMgr.init(this.map);
+
+  /**
+   * @type {boolean}
+   */
+  this.streetViewActive = true;
+
+  const streetViewToolActivate = new ngeoMiscToolActivate(this, 'streetViewActive');
+  ngeoToolActivateMgr.registerTool('mapTools', streetViewToolActivate, true);
+
+  /**
+   * @type {boolean}
+   */
+  this.dummyActive = false;
+
+  const dummyToolActivate = new ngeoMiscToolActivate(this, 'dummyActive');
+  ngeoToolActivateMgr.registerTool('mapTools', dummyToolActivate, false);
+}
+
+module.controller('MainController', MainController);
+
+module.constant('ngeoStreetviewOptions', {
+  viewer: 'mapillary',
+  key: 'TFI2dDVCMzRSeEd5MTVvOHVDOG93WDo0MGZkMTlmMThhNjIxODg5',
```

Comment:
@sbrunner I'm not sure we can publish this key in a public repo

Reply:
Effectively it shouldn't ...
Tthe option ngeoStreetviewOptions should be completely removed from here...